src/main/kotlin/nl/info/zac/admin/model/ZaaktypeCmmnBrpParameters.kt

```diff
@@ -38,11 +38,11 @@ class ZaaktypeCmmnBrpParameters {
     lateinit var zaaktypeCmmnConfiguration: ZaaktypeCmmnConfiguration
 
     @Column(name = "zoekWaarde")
-    var zoekWaarde: String? = ""
+    var zoekWaarde: String? = null
 
     @Column(name = "raadpleegWaarde")
-    var raadpleegWaarde: String? = ""
+    var raadpleegWaarde: String? = null
 
     @Column(name = "verwerkingsregisterWaarde")
-    var verwerkingsregisterWaarde: String? = ""
+    var verwerkingsregisterWaarde: String? = null
```

Comment:
I think this should be spelled without the connecting s, so:
```suggestion
    var verwerkingregisterWaarde: String? = null
```

Reply:
Good catch! Let's see if I can do that till we fix the other UI issue